lib/explorer/series.ex

```diff
@@ -6043,6 +6043,28 @@ defmodule Explorer.Series do
     end
   end
 
+  @doc """
+  Converts a string series containing valid json to a struct series.
+
+  ## Examples
+
+      iex> s = Series.from_list(["{\\"a\\":1}"])
+      iex> Series.json_decode(s, {:struct, %{"a" => {:s, 64}}})
+      #Explorer.Series<
+        Polars[1]
+        struct[1] [%{"a" => 1}]
+      >
+
+  Will raise `RuntimeError` for invalid json.
```

Comment:
Should we have the dytpe also be a list in case the json is a list?

Reply:
Let's also document happens if the dtype does not match.

Reply:
> Should we have the dytpe also be a list in case the json is a list?
Something like this ?
```elixir
s = Series.from_list(["[{\\"a\\":1}]"])
```

Reply:
Correct.